study.md

```diff
@@ -151,7 +155,13 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+GET - Used by browsers anytime a URL is is input. It's a request from the client for a read-only view of the resource.
+
+PUT - Used when you wanto to create or update the resource identified by the URL.
+
+DELETE - Request sent from client to delete the resource identified by the URL
+
+POST - Used when an update is required on the server side for the requested resource. A form is a good exmaple of this, where the data input by the user on the client side must be passed to the server side and stored.
```

Comment:
This is pretty close, POST creates a resource and PUT or PATCH updates or modifies the existing resource.